x-pack/plugins/security_solution/common/endpoint/schema/policy.ts

```diff
@@ -19,3 +20,21 @@ export const GetAgentPolicySummaryRequestSchema = {
     policy_id: schema.nullable(schema.string()),
   }),
 };
+
+const ListWithKuerySchema = schema.object({
+  page: schema.maybe(schema.number({ defaultValue: 1 })),
+  perPage: schema.maybe(schema.number({ defaultValue: 20 })),
+  sortField: schema.maybe(schema.string()),
```

Comment:
I think we agreed on `sort` for this https://github.com/elastic/security-team/issues/2149 though I don't mind this more explicit naming.